### ISSUE TITLE
Document Windows Installation Issue and Workaround

### ISSUE THREAD

**Issue by andrewkroh** [elastic/beats#540]
I am not sure where to document this, but I have run into this issue more than once so maybe it should be part of our Windows install documentation. I just encountered it while testing the install on XP. When running the `install-service-beat.ps1` if you see this error:

```
Windows PowerShell
Copyright (C) 2009 Microsoft Corporation. All rights reserved.

PS C:\Documents and Settings\Administrator> cd 'C:\Program Files\Winlogbeat'
PS C:\Program Files\Winlogbeat> .\install-service-winlogbeat.ps1
File C:\Program Files\Winlogbeat\install-service-winlogbeat.ps1 cannot be loaded because the execution of scripts is di
sabled on this system. Please see "get-help about_signing" for more details.
At line:1 char:33
+ .\install-service-winlogbeat.ps1 <<<<
    + CategoryInfo          : NotSpecified: (:) [], PSSecurityException
    + FullyQualifiedErrorId : RuntimeException
```

Then run this command to fix the issue.

```
PS C:\Program Files\Winlogbeat> Set-ExecutionPolicy RemoteSigned

Execution Policy Change
The execution policy helps protect you from scripts that you do not trust. Changing the execution policy might expose
you to the security risks described in the about_Execution_Policies help topic. Do you want to change the execution
policy?
[Y] Yes  [N] No  [S] Suspend  [?] Help (default is "Y"): Y
```

Now you can run this installation script:

```
PS C:\Program Files\Winlogbeat> .\install-service-winlogbeat.ps1

Status   Name               DisplayName
------   ----               -----------
Stopped  winlogbeat         winlogbeat
```

**Comment by andrewkroh** [comment]
On Windows 2012 R2 the workaround above didn't work for me. I used this:

```
PowerShell.exe -ExecutionPolicy UnRestricted -File .\install-service-packetbeat.ps1

Security warning
Run only scripts that you trust. While scripts from the internet can be useful, this script can potentially harm your
computer. If you trust this script, use the Unblock-File cmdlet to allow the script to run without this warning
message. Do you want to run C:\Program Files\Packetbeat\install-service-packetbeat.ps1?
[D] Do not run  [R] Run once  [S] Suspend  [?] Help (default is "D"): R

Status   Name               DisplayName
------   ----               -----------
Stopped  packetbeat         packetbeat
```

**Comment by dedemorton** [comment]
@andrewkroh I had to set my execution policy to unrestricted, too, to install on Windows 8. I'll add something to the installation doc to indicate that the user might need to set the execution policy to run the script. But I wonder why RemoteSigned works on some systems, but UnRestricted is required on others. I recall (vaguely) seeing something about the script not being signed. Can you add more detail so I get this right in the doc? Or I can explore further if you aren't sure....

**Comment by tsg** [comment]
Strange, I didn't have to do this on Windows 2012, but I always do "Run as Administrator". Maybe this is the reason?

**Comment by dedemorton** [comment]
It sounds like a common problem when you run PowerShell for the first time: http://www.howtogeek.com/106273/how-to-allow-the-execution-of-powershell-scripts-on-windows-7/. I'll add a note to the doc.

**Comment by andrewkroh** [comment]
We should probably recommend the following method because it does not force the users to make any permanent changes to their execution policy. This just sets the policy temporarily for the execution of the script.

`PowerShell.exe -ExecutionPolicy RemoteSigned -File .\install-service-packetbeat.ps1`

**Comment by andrewkroh** [comment]
Fixed in #559 and #560.
